server.go

```diff
@@ -274,7 +274,14 @@ func (s *server) channelHandleWrapper(handler ChannelHandler, handlerFunc Channe
 		// get the channel for this request - can be nil, e.g. FBA verification requests
 		channel, err := handler.GetChannel(ctx, r)
 		if err != nil {
-			logrus.WithError(err).WithField("request", string(recorder.Trace.RequestTrace))
+
+			// webhooks for FBA, IG and WAC should always return 200
+			if !handler.UseChannelRouteUUID() {
```

Comment:
I get that currently the 3 channels that want errors as 200s are the same 3 that don't use channel UUID routing... but feels like this could be more explicit... maybe handler defines a `ErrorResponseStatus() int` method and baseHandler returns 400?

Reply:
Could change `WriteRequestError` to take a status code, and then `WriteAndLogRequestError` should get the status code from the handler..